src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const obj = Object.assign({}, state);
```

Comment:
Good! but I want you to know that we can copy the object this way too `{ ...state }`